metrics/grafana/tikv_details.dashboard.py

```diff
@@ -5472,7 +5472,7 @@ def RocksDB() -> RowPanel:
             graph_panel(
                 title="Compaction operations",
                 description="The count of compaction and flush operations",
```

Comment:
maybe this helps distinguish the two metrics
```suggestion
                description="The rate of completed compaction and flush operations (left axis) and the count of running operations (right axis).",
```